data/hai/hai reveal 4 middle.txt

```diff
@@ -1412,33 +1412,180 @@ mission "Hai Reveal [A11] Sound the Alarm"
+					goto anger1
+			`	Nearly all Elders seem to agree, including the one who is questioning you.`
+			`	"During your time with Teeneep," he continues, "what was her main motive?"`
+				goto question2
+			label disbelief1
+			`	Most Elders shake their heads, but at least a third appears to agree with you. The Elder who questioned is increasingly annoyed, but continues calmly with another question.`
+			`	"During your time with Teeneep," he asks, "what was her main motive?"`
+				goto question2
+			label anger1
+			action
+				set "Hai Reveal: angered Elder in Council"
+			`	Most Elders in the room are shocked or offended, but a few nod in agreement with you. The Elder who questioned you looks furious.`
+			`	"During your subversive activities," the Elder says angrily, "what was Teeneep's main motive?" He grimaces as he spits out her name.`
+			label question2
+			choice
+				`	"Defending the innocent."`
+				`	"Opposing political stagnancy."`
+					label subversive2
+				`	"Advancing her political movement."`
+					label political2
+				`	"Saving the Hai from Elders like you who shirk their duty."`
+					label rage
+					to display
+						has "Hai Reveal: angered Elder in Council"
+				`	"Saving the Hai from Elders like you who shirk their duty."`
+					label anger2
+					to display
+						not "Hai Reveal: angered Elder in Council"
+			`	"Ah, one of the main talking points of her movement," he says, looking disappointed. "I see you've been brainwashed." At least a quarter of the Elders appear strongly offended by that. He shakes his head and walks away as another Elder approaches.`
+				goto question3
+			label subversive2
+			`	"The words of a child, Captain <last>," he says, disappointed. "Stability is the only sensible path forward. Anything else will just make matters worse." Most Elders you can see appear to agree with him, though often not wholeheartedly. He departs and another Elder approaches.`
+				goto question3
+			label political2
+			`	He shrugs with a look of understanding. "As do we all, I suppose. A foolish child she may be, but a politician nonetheless." He departs and another Elder approaches.`
+				goto question3
+			label anger2
+			action
+				set "Hai Reveal: angered Elder in Council"
+			`	He looks annoyed and scolds you. "Take care you don't end up like Teeneep, Captain <last>." After this, he returns to his seat and another Elder approaches you to speak.`
+				goto question3
+			label rage
+			action
+				set "Hai Reveal: threatened by Elder in Council"
+			`	The Elder stands angrily and smashes his fists upon a decorative railing, breaking it in half. "Take care you don't end up with a worse fate then Teeneep. The powers of an Elder can dispatch the unwary." He looks scared as he realizes what he just did. Most Elders are appalled, their functionaries are fearful, and the moderator scolds him in Hai. Guards escort him to a distant seat and another Elder approaches you to speak.`
```

Comment:
```suggestion
			`	The Elder shoots up angrily and smashes his fists upon a decorative railing, breaking it in half. "Take care you don't end up with a worse fate then Teeneep. The powers of an Elder can dispatch the unwary!" The Elder does the Hai equivalent of paling up as he realizes what he just did. Most Elders are appalled, their functionaries are fearful, and the moderator scolds him in Hai. Guards escort him to a distant seat and another Elder approaches you to speak.`
```
I wonder if the other Elders would start shouting (at him, at you, at Teeneep, whatever), instead of just being, not even looking, appalled. That sentence could be improved too. 

I chose to make him "pale up" to give him some more action and do more showing, but have you ever seen a pale squirrel? It's only an idea, and I might change it.

Reply:
I rewrote the paragraph with a better description of the event.